src/data/sym2.lean

```diff
@@ -453,4 +459,62 @@ end
 
 end decidable
 
+lemma filter_image_quotient_mk [decidable_eq α] (s : finset α) :
+  ((s.product s).image quotient.mk).filter (λ a : sym2 α, ¬a.is_diag) =
+    s.off_diag.image quotient.mk :=
```

Comment:
This might also have a good home next to where finset.off_diag is defined.

Reply:
off_diag itself could maybe earn it's own file

Reply:
How worthy do you think this lemma is? I introduced it to cut down a few lines from the proof, but it seems quite specialized to me. I don't think it's generalizable.

Reply:
I think it's a nice lemma to have since it links the definition of `off_diag` to `sym2.is_diag`.

Reply:
I don't think it's (currently?) possible to have this lemma closer to the def of `off_diag` because it really is about `sym2`.